README.md

```diff
@@ -189,8 +189,12 @@ This command will clone the `ros/meta-ros` repo into a subfolder under
 distro, commit them, and issue a pull request for `ros/meta-ros`. The
 `--ros-distro` flag must be supplied.
 
-If you don't want to issue a PR for `ros/meta-ros`, you should add
-the `--dry-run` flag. You can issue the PR later by using the `--pr-only` flag.
+Generating bitbake recipes without specifying `--dry-run` is not
```

Comment:
Drop [ ] from --dry-run in usage on line 145.

Add an optional `require_dryrun=False` parameter to `get_parser()`, which passes its value to the `required` parameter to the `add_argument()` invocation for `--dry-run` (see https://docs.python.org/3.6/library/argparse.html#required).